beginner/Exercise_7/Exercise_7.ipynb

```diff
@@ -216,45 +245,49 @@
    "id": "a5c93d5d",
    "metadata": {},
    "source": [
-    "### Create user token for https\n",
+    "### Create user token for HTTPS\n",
     "\n",
     "GitHub only allows authorized users to push to repositories.\n",
     "Therefore, we need to create a GitHub user token.\n",
     "\n",
-    "Please follow the [description](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token) provided by GitHub.\n",
+    "Please follow the description for [Creating a personal access token (classic)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic) provided by GitHub:\n",
```

Comment:
This is a bit confusing. The text and link suggest to create a classic token, however the description in the following refers to a fine-grained personal access token.